src/web/client/dal/remoteFetchProvider.ts

```diff
@@ -271,6 +271,31 @@ async function createContentFiles(
         }
 
         const attributeArray = attributes.split(",");
+
+        // Get rootpage id
+        let rootWebPageId = undefined;
+
+        if (isPortalVersionV2()) {
+            const content = entityDetails?.get(schemaEntityKey.CONTENT);
+
+            if (content) {
+                const contentValue = content ? result[content] : null;
+                if (contentValue === null) {
+                    throw new Error(ERRORS.CONTENT_EMPTY);
+                }
+                rootWebPageId = getRootPageId(contentValue);
+            }
+        } else if (isPortalVersionV1()) {
+            const rootpageidAttribute = entityDetails?.get(schemaEntityKey.ROOT_WEB_PAGE_ID);
+
+            if (rootpageidAttribute) {
+                rootWebPageId = rootpageidAttribute ? result[rootpageidAttribute] : null;
+                if (rootWebPageId === null) {
+                    throw new Error(ERRORS.ROOT_WEB_PAGE_ID_EMPTY);
```

Comment:
why do you want to block the whole page load if rootwebPageId is missing - though this will be conver most case and other things will be broken at this point - but Rule of Thumb for copresence specific code is dont block other flows. Just log an error. And make sure copresence code handles any missing data scenario.